src/reducers/notificationSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { INotification } from '@/interfaces/common';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface INotificationState extends INotification {
+  id: number;
+}
+const initialState: Array<INotificationState> = [];
+
+export const notificationSlice = createSlice({
+  name: 'notifications',
+  initialState,
+  reducers: {
+    setNotification(state, action: PayloadAction<INotification>) {
+      state.push({ id: state.length, ...action.payload });
+    },
+    clearNotification(state, action: PayloadAction<number>) {
+      return (state = state.filter((s) => s.id !== action.payload));
```

Comment:
Is it necessary to assign filtered resut into state variable? Please fix this

Reply:
Resolved